### PR TITLE
Remove brackets when loading to pairingList

### LINKED ISSUE #84: Square brackets in loading pairingfile log message

### PR DESCRIPTION
**Pull request by wilsonngja**
Fixes #84

### CODE REVIEW

File: src/main/java/seedu/duke/Storage.java [hunk 1]
```diff
@@ -209,16 +211,16 @@ public void loadPair(PairingList pairingList, File pairFile) {
                 String[] propertyParameters = pairingParameters[1].split("\\s\\|\\s");
 
                 //Client Information
-                String clientName = clientParamters[0];
+                String clientName = clientParamters[0].replace(OPEN_BRACKET, EMPTY_STRING);
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/Storage.java:214]: Good use of Java's String API to do the string manipulation for you, it lowers chances of bugs.

Comment (ngdeqi) [inline src/main/java/seedu/duke/Storage.java:214]: Good use of Java's String API to do the string manipulation for you, it lowers chances of bugs.

**Review by ngdeqi — COMMENTED** [review]
LGTM

File: src/main/java/seedu/duke/Storage.java (hunk 1 above)
Comment (FeliciaBeatrice) [inline src/main/java/seedu/duke/Storage.java:214]: can change clientParamters to clientParameters to make it consistent with the others

**Review by OVReader — APPROVED** [review]
All good, minor typo will be rectified in subsequent PR

**Pull request merged**